Dockerfile cannot be built

It looks like Ruby fails to build. The relevant output from `docker build` is:
```
Step 13/42 : RUN echo 'gem: --no-document' >> /usr/local/etc/gemrc &&    mkdir /src && cd /src && git clone https://github.com/sstephenson/ruby-build.git &&    cd /src/ruby-build && ./install.sh &&    cd / && rm -rf /src/ruby-build && ruby-build $RUBY_VERSION /usr/local
 ---> Running in 1b68e5430c29
Cloning into 'ruby-build'...
Downloading ruby-2.3.1.tar.bz2...
-> https://cache.ruby-lang.org/pub/ruby/2.3/ruby-2.3.1.tar.bz2
Installing ruby-2.3.1...

BUILD FAILED (Debian GNU/Linux 9 using ruby-build 20181019)

Inspect or clean up the working tree at /tmp/ruby-build.20181130172419.22
Results logged to /tmp/ruby-build.20181130172419.22.log

Last 10 log lines:
make[2]: Leaving directory '/tmp/ruby-build.20181130172419.22/ruby-2.3.1/ext/openssl'
exts.mk:208: recipe for target 'ext/openssl/all' failed
make[1]: *** [ext/openssl/all] Error 2
make[1]: *** Waiting for unfinished jobs....
installing default nkf libraries
linking shared-object nkf.so
make[2]: Leaving directory '/tmp/ruby-build.20181130172419.22/ruby-2.3.1/ext/nkf'
make[1]: Leaving directory '/tmp/ruby-build.20181130172419.22/ruby-2.3.1'
uncommon.mk:203: recipe for target 'build-ext' failed
make: *** [build-ext] Error 2
The command '/bin/sh -c echo 'gem: --no-document' >> /usr/local/etc/gemrc &&    mkdir /src && cd /src && git clone https://github.com/sstephenson/ruby-build.git &&    cd /src/ruby-build && ./install.sh &&    cd / && rm -rf /src/ruby-build && ruby-build $RUBY_VERSION /usr/local' returned a non-zero code: 1
```
@vbrown608, can you take a look at this when you get a chance? I played around with it for a few minutes and thought the fix might be as simple as installing openssl but that didn't work.

```
Step 13/42 : RUN echo 'gem: --no-document' >> /usr/local/etc/gemrc &&    mkdir /src && cd /src && git clone https://github.com/sstephenson/ruby-build.git &&    cd /src/ruby-build && ./install.sh &&    cd / && rm -rf /src/ruby-build && ruby-build $RUBY_VERSION /usr/local
 ---> Running in b3c480102c7f
Cloning into 'ruby-build'...
Downloading ruby-2.3.1.tar.bz2...
-> https://cache.ruby-lang.org/pub/ruby/2.3/ruby-2.3.1.tar.bz2
Installing ruby-2.3.1...

BUILD FAILED (Debian GNU/Linux 9 using ruby-build 20190401)

Inspect or clean up the working tree at /tmp/ruby-build.20190415195431.23
Results logged to /tmp/ruby-build.20190415195431.23.log

Last 10 log lines:
make[2]: Leaving directory '/tmp/ruby-build.20190415195431.23/ruby-2.3.1/ext/json/generator'
linking shared-object bigdecimal.so
make[2]: Leaving directory '/tmp/ruby-build.20190415195431.23/ruby-2.3.1/ext/bigdecimal'
linking shared-object date_core.so
make[2]: Leaving directory '/tmp/ruby-build.20190415195431.23/ruby-2.3.1/ext/date'
linking shared-object nkf.so
make[2]: Leaving directory '/tmp/ruby-build.20190415195431.23/ruby-2.3.1/ext/nkf'
make[1]: Leaving directory '/tmp/ruby-build.20190415195431.23/ruby-2.3.1'
uncommon.mk:203: recipe for target 'build-ext' failed
make: *** [build-ext] Error 2
ERROR: Service 'site' failed to build: The command '/bin/sh -c echo 'gem: --no-document' >> /usr/local/etc/gemrc &&    mkdir /src && cd /src && git clone https://github.com/sstephenson/ruby-build.git &&    cd /src/ruby-build && ./install.sh &&    cd / && rm -rf /src/ruby-build && ruby-build $RUBY_VERSION /usr/local' returned a non-zero code: 1
```

--- a/Dockerfile
+++ b/Dockerfile
@@ -21,7 +21,7 @@ ENV LC_ALL C.UTF-8
 # need rsync for deploy script and texlive for building docs
 RUN apt-get install -y --no-install-recommends \
     imagemagick \
-    gsfonts-x11 \
+    gsfonts \
     latexmk \
     rsync \
     texlive \

--- a/Dockerfile
+++ b/Dockerfile
@@ -21,6 +21,7 @@ ENV LC_ALL C.UTF-8
 # need rsync for deploy script and texlive for building docs
 RUN apt-get install -y --no-install-recommends \
     imagemagick \
+    gsfonts-x11 \
     latexmk \
     rsync \
     texlive \

